src/EditorFeatures/Core/Implementation/Diagnostics/DiagnosticsSquiggleTaggerProvider.cs

```diff
@@ -23,21 +23,20 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.Diagnostics
     [TagType(typeof(IErrorTag))]
     internal partial class DiagnosticsSquiggleTaggerProvider : AbstractDiagnosticsAdornmentTaggerProvider<IErrorTag>
     {
-        private readonly bool _blueSquiggleForBuildDiagnostic;
-
         private static readonly IEnumerable<Option<bool>> s_tagSourceOptions =
             ImmutableArray.Create(EditorComponentOnOffOptions.Tagger, InternalFeatureOnOffOptions.Squiggles, ServiceComponentOnOffOptions.DiagnosticProvider);
+
         protected internal override IEnumerable<Option<bool>> Options => s_tagSourceOptions;
 
+        private bool? _blueSquiggleForBuildDiagnostic;
```

Comment:
If you use it, we'll keep it.  But i'm not a fan of us having code around that no customers will use.